scripts/start.sh

```diff
@@ -48,6 +48,24 @@ sleep 5
 echo "Creating role bindings for principals"
 docker-compose exec tools bash -c "/tmp/helper/create-role-bindings.sh" || exit 1
 
+# Workaround for setting min ISR on topic _confluent-metadata-auth
+docker-compose exec kafka1 kafka-configs \
+   --bootstrap-server kafka1:12091 \
+   --entity-type topics \
+   --entity-name _confluent-metadata-auth \
+   --alter \
+   --add-config min.insync.replicas=1
+
+# Create Kafka topic users, using appSA principal
+docker-compose exec kafka1 bash -c 'export KAFKA_LOG4J_OPTS="-Dlog4j.rootLogger=DEBUG,stdout -Dlog4j.logger.kafka=DEBUG,stdout" && kafka-topics \
```

Comment:
Are these log4j options needed?

Reply:
@mikebin Without the log4j options, the output includes `“WARN The configuration ... was supplied but isn’t a known config. ”` so it's not required per se, but it generates cleaner output.  WDYT?

Reply:
Sounds fine to me. I'm curious why setting `DEBUG` _reduces_ the amount of logging instead of increasing it, though.